calour/export_html_template.html

```diff
@@ -61,9 +61,9 @@
       var yaxiswidth = 100;
       var xaxisheight = 20;
 
-      // fix for safari image smoothing interpolation - we want it pixelated!
-      var ctx = document.getElementById('canvas').getContext('2d');
-      ctx.webkitImageSmoothingEnabled = false;
+      // // fix for safari image smoothing interpolation - we want it pixelated!
```

Comment:
could you add comments? or just deleted it if not relevant any more.

Reply:
not relevant. removed